code/modules/mob/living/simple_animal/hostile/boss/boss.dm

```diff
@@ -0,0 +1,8 @@
+// Only here to serve as a filter, say you don't want boss mobs to spawn from gold slimes
```

Comment:
Did you make sure you can't sentience potion it though (you guys have those right?)

You can also cheese this by using a bomb, supermatter, wand of death, etc to instakill it